### PR TITLE
COR-605: email validation, update error message

### PR DESCRIPTION
**Pull request by internetti**
this is a tentative PR, since I find date validation to be quite unreliable, it sometimes works, sometimes not, I do not see a pattern.
everything else already works.

I would like to present the error in a nicer way, could do that in a different PR once I figure out how

### CODE REVIEW

File: internal/api/utils.go [hunk 1]
```diff
@@ -9,11 +10,14 @@ var dateFormat = "2006-01-02"
 
 func ParseDate(s string) (*time.Time, error) {
 	if s != "" {
-		birthDate, err := time.Parse(dateFormat, s)
+		date, err := time.Parse(dateFormat, s)
+		if date.IsZero() {
+			return nil, fmt.Errorf("date is zero, %s", date)
+		}
```
Comment (neb42) [inline internal/api/utils.go:16]: Should this go after the `err` check?

**Comment by neb42** [comment]
lgmt, but you've also said this is "tentative". I'd say add a do not merge label if it still is.

**Comment by internetti** [comment]
that shouldn't be necessary since we decided that the reviewee merges the PRs, but doesn't hurt.
But it's only tentative because I don't know if anyone can confirm this unreliable behaviour, and if I therefore still need to fix this. can you confirm that you can sometimes upload invalid birthdays? as in format DD.MM.YYYY for example? works like every third time for me. no idea why.

**Comment by neb42** [comment]
> that shouldn't be necessary since we decided that the reviewee merges the PRs, but doesn't hurt.

I think it's good practice. If someone goes on holiday, I would potentially look at merging their prs.

> I don't know if anyone can confirm this unreliable behaviour

Once prs are merged and main is stable I'm going to do some qa, so I can check then

**Pull request merged**